app/templates/components/datafruits-chat.hbs

```diff
@@ -34,33 +34,52 @@
     </form>
   {{else}}
     <form align="center" class="mb-3">
-      <label for="nick">{{t "chat.label.nick"}}</label>
-      <Input
-        @type="text"
-        @value={{nick}}
-        @autocorrect="off"
-        @autocapitalize="none"
-        class=""
-        placeholder={{t "chat.nickname"}}
-      />
-      <button {{action "enterChatAnonymously"}}
-        class="cool-button"
-        disabled={{or disableJoinButton networkStatus.isOffline}}
-        title={{t "chat.titles.enter"}}
-      >
-        {{t "chat.join_anonymously"}}
-      </button>
-      <button
-        title={{if this.gifsEnabled  (t "chat.titles.img_on") (t "chat.titles.img_off") }}
-        class="cool-button mt-1 md:m-0" {{action "toggleGifsEnabled"}}>
-        {{t (if this.gifsEnabled  "chat.img_on" "chat.img_off") }}
-      </button>
-      <div class="flex">
+      <div>
+        <div>
+          <label for="nick">{{t "chat.label.nick"}}</label>
+          <Input
+            @type="text"
+            @value={{nick}}
+            @autocorrect="off"
+            @autocapitalize="none"
+            class="max-h-full"
+            placeholder={{t "chat.nickname"}}
+          />
+          <button {{action "enterChatAnonymously"}}
+            class="cool-button"
+            disabled={{disableJoinButton}}
+            title={{t "chat.titles.enter"}}
+          >
+            {{t "chat.join_anonymously"}}
+          </button>
+          <button
+            title={{if this.gifsEnabled  (t "chat.titles.img_on") (t "chat.titles.img_off") }}
+            class="cool-button mt-1 md:m-0" {{action "toggleGifsEnabled"}}>
+            {{t (if this.gifsEnabled  "chat.img_on" "chat.img_off") }}
+          </button>
+        </div>
+        <div>
+          <Input
+            name="agree-to-coc"
+            id="agree-to-coc"
+            @type="checkbox"
+            @checked={{this.agreeToCoC}}
+            aria="Agree with the Code of Conduct to join the chat"
```

Comment:
```suggestion
            aria={{t "coc.checkbox_aria"}}
```
or something

Reply:
Ah woops, yeah that should be a translation, not  a plain string